From 481457db4fe870fcbab47fe29cb768daf235fd1f Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Thu, 13 Oct 2011 12:17:27 +0200
Subject: [PATCH 2/2] am335x_evm: set bootdelay to 1

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 include/configs/am335x_evm.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index 9ce360d..78c823b 100755
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -38,7 +38,7 @@
 #define CONFIG_INITRD_TAG		/* Required for ramdisk support */
 
 /* set to negative value for no autoboot */
-#define CONFIG_BOOTDELAY		3
+#define CONFIG_BOOTDELAY		1
 
 #define CONFIG_MMC
 #define CONFIG_NAND
-- 
1.6.6.1

