From 527309ca11da0a258479da8443f00de0003c4bf9 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Wed, 12 Oct 2011 10:50:51 +0200
Subject: [PATCH 2/2] am335x_evm: read uEnv.txt instead of boot.scr

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 include/configs/am335x_evm.h |   25 ++++++++++++++-----------
 1 files changed, 14 insertions(+), 11 deletions(-)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index dbb412a..5a28724 100755
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -116,6 +116,10 @@
 	"loadbootscript=fatload mmc 0 ${script_addr} boot.scr\0" \
 	"bootscript= echo Running bootscript from MMC/SD to set the ENV...; " \
 		"source ${script_addr}\0" \
+	"bootenv=uEnv.txt\0" \
+	"loadbootenv=fatload mmc 0 ${loadaddr} ${bootenv}\0" \
+	"importbootenv=echo Importing environment from mmc ...; " \
+			"env import -t $loadaddr $filesize\0" \
 	"mmc_load_uimage_fat=fatload mmc 0 ${loadaddr} ${bootfile}\0" \
 	"mmc_load_uimage=ext2load mmc 0:2 ${loadaddr} /boot/${bootfile}\0" \
 	"bootargs_defaults=setenv bootargs " \
@@ -167,18 +171,17 @@
 
 #define CONFIG_BOOTCOMMAND \
 	"if mmc rescan; then " \
-		"if run loadbootscript; then " \
-			"run bootscript; " \
-		"else " \
-			"if run mmc_load_uimage; then " \
-				"run mmc_boot; " \
-			"else " \
-				"run nand_boot; " \
-			"fi; " \
+		"if run loadbootenv; then " \
+			"echo Loaded environment from ${bootenv};" \
+			"run importbootenv; " \
 		"fi; " \
-	"else " \
-		"run nand_boot; " \
-	"fi"
+		"if test -n $uenvcmd; then " \
+			"echo Running uenvcmd ...;" \
+			"run uenvcmd;" \
+		"fi;" \
+		"run mmc_boot; " \
+	"fi; " \
+	"run nand_boot; " \
 
 #define CONFIG_DISPLAY_BOARDINFO
 #define CONFIG_MISC_INIT_R
-- 
1.6.6.1

