From b01ee680bab7b40f6dc9b8087630e5c0251c6fcd Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Sun, 1 Apr 2012 23:10:22 +0200
Subject: [PATCH 08/10] am335x-evm: load uImage from /boot instead of VFAT

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 include/configs/am335x_evm.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index a81d716..e42febe 100755
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -101,7 +101,7 @@
 		"nfsroot=${serverip}:${rootpath},${nfsopts} rw " \
 		"ip=dhcp\0" \
 	"mmc_boot=run mmc_args; " \
-		"run mmc_load_uimage; " \
+		"run mmc_load_uimage_ext2; " \
 		"bootm ${kloadaddr}\0" \
 	"nand_boot=echo Booting from nand ...; " \
 		"run nand_args; " \
@@ -139,7 +139,7 @@
 			"echo Running uenvcmd ...;" \
 			"run uenvcmd;" \
 		"fi;" \
-		"if run mmc_load_uimage; then " \
+		"if run mmc_load_uimage_ext2; then " \
 			"run mmc_args;" \
 			"bootm ${kloadaddr};" \
 		"fi;" \
-- 
1.7.10

