From 9e1a99476875b1f1e6e34e70d9bf6e77d3ac8fba Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Wed, 4 Apr 2012 00:06:31 +0200
Subject: [PATCH 10/10] am335x: switch to ext4 mode

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 include/configs/am335x_evm.h |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index e42febe..e2be380 100755
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -75,7 +75,7 @@
 	"importbootenv=echo Importing environment from mmc ...; " \
 		"env import -t $loadaddr $filesize\0" \
 	"mmc_load_uimage=fatload mmc ${mmc_dev}:1 ${kloadaddr} ${bootfile}\0" \
-	"mmc_load_uimage_ext2=ext2load mmc ${mmc_dev}:2 ${kloadaddr} /boot/${bootfile}\0" \
+	"mmc_load_uimage_ext4=ext4load mmc ${mmc_dev}:2 ${kloadaddr} /boot/${bootfile}\0" \
 	"bootargs_defaults=setenv bootargs " \
 		"console=${console} " \
 		"${optargs}\0" \
@@ -101,7 +101,7 @@
 		"nfsroot=${serverip}:${rootpath},${nfsopts} rw " \
 		"ip=dhcp\0" \
 	"mmc_boot=run mmc_args; " \
-		"run mmc_load_uimage_ext2; " \
+		"run mmc_load_uimage_ext4; " \
 		"bootm ${kloadaddr}\0" \
 	"nand_boot=echo Booting from nand ...; " \
 		"run nand_args; " \
@@ -139,7 +139,7 @@
 			"echo Running uenvcmd ...;" \
 			"run uenvcmd;" \
 		"fi;" \
-		"if run mmc_load_uimage_ext2; then " \
+		"if run mmc_load_uimage_ext4; then " \
 			"run mmc_args;" \
 			"bootm ${kloadaddr};" \
 		"fi;" \
@@ -426,6 +426,7 @@
 #define CONFIG_DOS_PARTITION
 #define CONFIG_CMD_FAT
 #define CONFIG_CMD_EXT2
+#define CONFIG_CMD_EXT4
 #endif
 
 /* Unsupported features */
-- 
1.7.10

