From b2ed58548de12a9bd9b23f51ade86793ae44b74f Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Mon, 11 Jul 2011 09:47:17 +0200
Subject: [PATCH 35/35] omap4_panda: use uImage.panda for tftp

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 include/configs/omap4_panda.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/configs/omap4_panda.h b/include/configs/omap4_panda.h
index 0fcdf46..454a6b0 100644
--- a/include/configs/omap4_panda.h
+++ b/include/configs/omap4_panda.h
@@ -132,7 +132,7 @@
 #define CONFIG_NET_MULTI		1
 #define CONFIG_CMD_PING			1
 #define CONFIG_CMD_DHCP			1
-#define CONFIG_BOOTFILE			"uImage"
+#define CONFIG_BOOTFILE			"uImage.panda"
 
 /* Flash */
 #define CONFIG_SYS_NO_FLASH	1
-- 
1.6.6.1

