From 8c7d293dcbf1bb53c4e8f26fc2553e99880978d1 Mon Sep 17 00:00:00 2001
From: Jason Kridner <jdk@ti.com>
Date: Sat, 8 Oct 2011 15:48:57 -0400
Subject: [PATCH 2/5] am335x_evm: reduced bootdelay to 0

---
 include/configs/am335x_evm.h |    5 +----
 1 files changed, 1 insertions(+), 4 deletions(-)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index 0f806aa..c5e608c 100755
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -24,7 +24,7 @@
 #define CONFIG_AM335X_HSMMC_INSTANCE	0	/* 0 - MMC0, 1 - MMC1 */
 
 /* set to negative value for no autoboot */
-#define CONFIG_BOOTDELAY		3
+#define CONFIG_BOOTDELAY		0
 
 # if defined(CONFIG_SPI_BOOT)
 #  define CONFIG_SPI			1
@@ -73,9 +73,6 @@
 #define CONFIG_SETUP_MEMORY_TAGS	1
 #define CONFIG_INITRD_TAG		1 /* Required for ramdisk support */
 
-/* set to negative value for no autoboot */
-#define CONFIG_BOOTDELAY		3
-
 #define CONFIG_MMC			1
 
 #ifndef CONFIG_NOR_BOOT
-- 
1.6.6.1

