From 9de8f53dea91169f3b8a9473ad2b5c9c0fdd0eb1 Mon Sep 17 00:00:00 2001
From: Jason Kridner <jdk@ti.com>
Date: Sat, 8 Oct 2011 16:22:53 -0400
Subject: [PATCH 3/5] am335x_evm: boot kernel from ext2 filesystem

---
 include/configs/am335x_evm.h |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index c5e608c..a2e4564 100755
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -112,7 +112,8 @@
 	"loadbootscript=fatload mmc 0 ${script_addr} boot.scr\0" \
 	"bootscript= echo Running bootscript from MMC/SD to set the ENV...; " \
 		"source ${script_addr}\0" \
-	"mmc_load_uimage=fatload mmc 0 ${loadaddr} ${bootfile}\0" \
+	"mmc_load_uimage_fat=fatload mmc 0 ${loadaddr} ${bootfile}\0" \
+	"mmc_load_uimage=ext2load mmc 0:2 ${loadaddr} /boot/${bootfile}\0" \
 	"bootargs_defaults=setenv bootargs " \
 		"console=${console}\0 " \
 	"mmc_args=run bootargs_defaults;" \
@@ -403,6 +404,7 @@
 # define CONFIG_CMD_MMC		1
 # define CONFIG_DOS_PARTITION	1
 # define CONFIG_CMD_FAT		1
+# define CONFIG_CMD_EXT2	1
 #endif
 
 /* Unsupported features */
-- 
1.6.6.1

