From 86e89bb160cfe67c173f1260b93066b12dd73580 Mon Sep 17 00:00:00 2001
From: Enrico Butera <ebutera@users.berlios.de>
Date: Thu, 22 Dec 2011 11:03:58 +0100
Subject: [PATCH 2/2] fix toolchain names in config.bld

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 .../dmai/packages/config.bld                       |   24 +------------------
 1 files changed, 2 insertions(+), 22 deletions(-)

diff --git a/davinci_multimedia_application_interface/dmai/packages/config.bld b/davinci_multimedia_application_interface/dmai/packages/config.bld
index 8c2e96e..566f907 100644
--- a/davinci_multimedia_application_interface/dmai/packages/config.bld
+++ b/davinci_multimedia_application_interface/dmai/packages/config.bld
@@ -44,28 +44,8 @@ var C6X = xdc.useModule('ti.targets.C64P');
 C6X.rootDir = codegen;
 C6X.platform = xdcplat;
 
-/* User passes in $(CROSS_COMPILE) where $(CROSS_COMPILE)gcc is their compiler
-   Then the TOOLDIR and LONGNAME are derived based on a regex of CROSS_COMPILE
-*/
-var crosscompile = "" + java.lang.System.getenv("CROSS_COMPILE");
-
-var tooldir = "";
-var longName = "";
-
-/* Search CROSS_COMPILE for bin/ If only 1 bin/ is found, set the tooldir to
-   the path prior to bin/ and the prefix to "bin/" + remainder of path,
-   else leave the tooldir as "" and set the LONGNAME to the full CROSS_COMPILE
-   path
-*/
-var regex = new RegExp("bin/");
-var find = crosscompile.split( regex );
-
-if (find[0]!=crosscompile && find.length==2) {
-    tooldir = find[0];
-    longName = "bin/" + find[1] + "gcc";
-} else {
-    longName = crosscompile + "gcc";
-}
+var tooldir = "" + java.lang.System.getenv("CSTOOL_DIR");
+var longName = "" + java.lang.System.getenv("MVTOOL_PREFIX") + "gcc";
 
 /* location of the GCC Arm9 tools */
 var GCArmv5T = xdc.useModule('gnu.targets.arm.GCArmv5T');
-- 
1.7.2.5

