From 6133415cf067d656c40997e00eace06d07aeab34 Mon Sep 17 00:00:00 2001
From: Eyal Reizer <eyalr@ti.com>
Date: Mon, 16 May 2011 17:47:46 +0300
Subject: [PATCH] PSP04.02.00.07.sdk: activate wireless extensions interface used by iwconfig

Signed-off-by: Eyal Reizer <eyalr@ti.com>
---
 arch/arm/mach-omap2/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-omap2/Kconfig b/arch/arm/mach-omap2/Kconfig
index dfa5464..8152b72 100644
--- a/arch/arm/mach-omap2/Kconfig
+++ b/arch/arm/mach-omap2/Kconfig
@@ -34,6 +34,7 @@ config ARCH_OMAP3
 	select ARM_L1_CACHE_SHIFT_6 if !ARCH_OMAP4
 	select ARCH_HAS_OPP
 	select PM_OPP if PM
	select WL12XX_PLATFORM_DATA
+	select WIRELESS_EXT
 
 config ARCH_OMAP4
 	bool "TI OMAP4"
-- 
1.7.0.4

